sdk/CMakeLists.txt

```
@@ -87,23 +87,35 @@ if(EXECUTORCH_SEPARATE_FLATCC_HOST_PROJECT)
   # Add the host project. We build this separately so that we can generate
   # headers on the host during the build, even if we're cross-compiling the
   # flatcc runtime to a different architecture.
-
-  # lint_cmake: -readability/wonkycase
-  ExternalProject_Add(
-    flatcc_project
-    PREFIX ${CMAKE_BINARY_DIR}/_host_build
-    SOURCE_DIR ${_flatcc_source_dir}
-    BINARY_DIR ${CMAKE_BINARY_DIR}/_host_build
-    CMAKE_CACHE_ARGS
-      -DFLATCC_TEST:BOOL=OFF -DFLATCC_REFLECTION:BOOL=OFF
-      # See above comment about POSITION_INDEPENDENT_CODE.
-      -DCMAKE_POSITION_INDEPENDENT_CODE:BOOL=ON
-    INSTALL_COMMAND "" # Prevent the install step
+  execute_process(
+    COMMAND ${CMAKE_COMMAND} ${_flatcc_source_dir}
+    -DFLATCC_TEST=OFF -DFLATCC_REFLECTION=OFF -DCMAKE_POSITION_INDEPENDENT_CODE=ON
+    -B${CMAKE_BINARY_DIR}/_host_build
```

One last request: please wrap this to fit in 80 columns (to satisfy an internal lint checker), and please restore the comment that was here before.

```suggestion
    -DFLATCC_TEST=OFF -DFLATCC_REFLECTION=OFF
    # See above comment about POSITION_INDEPENDENT_CODE.
    -DCMAKE_POSITION_INDEPENDENT_CODE=ON
    -B${CMAKE_BINARY_DIR}/_host_build
```

---

Thank you, is the `cmake-format` used for linter? We'll keep in mind for other changes.

---

If you run cmake-format, then the linters will be happy. Unfortunately, right now we have different linters in github (lintrunner) and inside of Meta, so they don't always agree, and there's no way for you to see the Meta-internal linters. We plan to disable the internal linters so that lintrunner in github is the source of truth.

Thanks for making this final fix! I'll try landing again today.